code/game/area/Space Station 13 areas.dm

```diff
@@ -2582,6 +2582,22 @@ area/security/podbay
 /area/awaycontent/a30
 	icon_state = "awaycontent30"
 
+////////////////////////VR AREAS///////////////////////////////////
+
+/area/vr
+	name = "VR"
+	requires_power = 0
+	dynamic_lighting = 0
+	no_teleportlocs = 1

```

Comment:
Is this going to be on a new zlevel? If it is, this is arguably redundant - just block teleporting on the whole zlevel?
If it isn't, then how to you prevent someone finding the VR area by flying in space?